settings/nitrofiles/languages/english.ini

```diff
@@ -53,17 +53,17 @@ SLOW=Slow
 
 DESCRIPTION_FRAMERATE=Changes how many frames per second are displayed.
 DESCRIPTION_DSCLASSICMENU=The menu that is shown by pressing SELECT in the DSi theme, can be shown before the ROM select menu.
-DESCRIPTION_SELECTBUTTONOPTION=Choose a menu to show, when pressing SELECT in the DSi or SEGA Saturn theme.
+DESCRIPTION_SELECTBUTTONOPTION=Choose a menu to show when pressing SELECT in the DSi or SEGA Saturn theme.
 DESCRIPTION_THEME_1=The theme to use in TWiLight Menu++. Press Left/Right to select, \A for skins.
 DESCRIPTION_DSIMUSIC=Music to use in the DSi and 3DS themes.
```

Comment:
Select which song should be played in the Nintendo DSi, Nintendo 3DS, SEGA Saturn and Homebrew Launcher themes.